src/platform/utilities/feature-toggles/featureFlagNames.js

```diff
@@ -3,6 +3,7 @@ export default Object.freeze({
   dashboardShowCovid19Alert: 'dashboardShowCovid19Alert',
   facilityLocatorShowCommunityCares: 'facilityLocatorShowCommunityCares',
```

Comment:
I don't like leaving both flags in the code, as this creates confusion. I think we should at least add a comment to explain that `facilityLocatorShowCommunityCares` is deprecated, and reach out to the teams that are currently using it.

Reply:
Yeah I don't think we should remove the flag now but adding a comment.